src/App.jsx

```diff
@@ -13,12 +13,16 @@ import Login from './components/Login';
 import Questions from './components/Questions';
 import DebutJeu from './components/DebutJeu';
 import PageScan from './components/PageScan';
+import ChoixQuestion from './components/ChoixQuestion';
+import Réponse from './components/Réponse';
+
 
 export default function App() {
   return (
     <div className="AppContent">
       <Router>
         <Switch>
+          <Route path="/reponse" component={Réponse} />
```

Comment:
no special characters like é in component name

your urls are in english, so it should be /answer (/answers/:answerId ?) and Answer

Reply:
i will fix the special caracter but wi will change all our french adress in the next project session